subprojects/smoke-test/src/smokeTest/groovy/org/gradle/smoketests/AndroidSantaTrackerSmokeTest.groovy

```diff
@@ -50,9 +50,6 @@ class AndroidSantaTrackerSmokeTest extends AbstractSmokeTest {
 
         expect:
         expectDeprecationWarnings(result,
-            "Property 'excludeListProvider' is not annotated with an input or output annotation. This behaviour has been deprecated and is scheduled to be removed in Gradle 7.0.",
```

Comment:
Why don't we have a type for this `Property`? Where is it declared (has it been declared)?

Reply:
This message is (was) printed right underneath the name of the task that has the property. It's not ideal, as you still need to figure out the type of the task (and which plugin it belongs to), but any other approach I tried to display the type name led to too verbose outputs.